.github/dependabot.yml

```diff
@@ -19,3 +19,25 @@ updates:
   target-branch: ci/dependabot-updates
   labels:
     - CI/CD
+
+# For pydantic v1 support branch - Should be removed after this is no longer being
+# supported (end of 2023)
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: weekly
+    day: monday
+    time: "04:18"
```

Comment:
consider making a comment saying that time are selected arbitrarily [but slightly differently]

Reply:
Definitely make a note that you want to avoid 'popular' hours